opendbc/dbc/generator/honda/_nidec_scm_group_b.dbc

```diff
@@ -5,6 +5,8 @@ BO_ 662 SCM_BUTTONS: 4 SCM
  SG_ COUNTER : 29|2@0+ (1,0) [0|3] "" EON
 
 BO_ 806 SCM_FEEDBACK: 8 SCM
```

Comment:
This is another one we should consolidate, _nidec_scm_group_b is the same thing as Bosch, needs a round of generator cleanup. Same as KINEMATICS, just got done doing a bunch of generator cleanup and want to avoid adding new drift/rot where every car's version of that message ends up being slightly different. I'll do the cleanup myself at some point if you don't have time.